packages/preview/supercharged-dhbw/3.0.0/template/main.typ

```diff
@@ -0,0 +1,130 @@
+#import "@local/supercharged-dhbw:3.0.0": *
```

Comment:
Can you please update this import to use the `@preview` namespace?

Reply:
Thanks for pointing that out 🙏🏻

I forgot to change it back 🙂